test/splitWith.js

```diff
@@ -0,0 +1,21 @@
+var R = require('../source');
+var eq = require('./shared/eq');
+
+
+describe('splitWith', function() {
+  it('splits an array at every instance of predicate matching', function() {
+    eq(R.splitWith(R.equals(2), [1, 2, 3, 4, 2, 1]), [[1], [3, 4], [1]]);
+  });
+  it('retains all original elements', function() {
+    eq(R.splitWith(R.equals(2), [1, 1, 1]), [[1, 1, 1]]);
+  });
```

Comment:
`'splitWith' retains all original elements` doesn't tell me anything.  You need another clause.  Perhaps, `when the predicate matches nothing.`